src/Microsoft.Extensions.Globalization.CultureInfoCache/project.json

```diff
@@ -1,6 +1,10 @@
 {
   "version": "1.0.0-*",
   "description": "Provides cached instances of CultureInfo using a generated list of known culture names for use in scenarios where unbounded CultureInfo creation is undesirable.",
+  "tags": [
+    "globalization",
```

Comment:
Localization project.json also has "aspnetcore" in the tags. Consider adding it here too.